{"n_dim": 55, "n_trials": 130, "objective": "markowitz_analytic_on_cube", "white": "scipy_powell", "black": "dlib", "traceback": ["passing", "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/humpday/comparison/eloratings.py\", line 76, in optimizer_game\n    black_best_val, black_best_x, black_feval_count = black(objective, n_trials=n_black_trials,\n  File \"/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/humpday/optimizers/dlibcube.py\", line 32, in dlib_cube\n    return dlib_default_cube(objective=objective, n_trials=n_trials, n_dim=n_dim, with_count=with_count)  # It is useful to have a clone of one of the better algos\n  File \"/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/humpday/optimizers/dlibcube.py\", line 27, in dlib_default_cube\n    best_x, best_val = find_min_global(_objective, lb, ub, n_trials)\nRuntimeError: \n\nError detected at line 63.\nError detected in file /tmp/pip-install-xg7_mpy4/dlib_a689e4838da64a53bc76a2b78e391f07/tools/python/src/global_optimization.cpp.\nError detected in function double call_func(pybind11::object, const dlib::matrix<double, 0, 1>&).\n\nFailing expression was 0 < num && num < 35.\nFunctions being optimized must take between 1 and 35 scalar arguments.\n\n"], "best_val": [1.9896003049705373, null], "best_x": [[0.9999338930386481, 0.5213113847429129, 0.49803102050016024, 0.49458921080540663, 0.15975542163275785, 6.610696135189609e-05, 0.13466236360207562, 0.022151880114302795, 6.610696135189609e-05, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], null], "feval_count": [131, null], "n_trials_instructed": [130, 131], "passing": [true, false], "completed": false, "winner": "incomplete", "points": null}